lib/ecmascript.ts

```diff
@@ -122,7 +121,7 @@ const ALLOWED_UNITS = [
   'microsecond',
   'nanosecond'
 ];
-const SINGULAR_PLURAL_UNITS = [
+const SINGULAR_PLURAL_UNITS: Array<[string, string]> = [
```

Comment:
`ReadonlyArray<readonly [string, string]>` ?

What about just adding `as const` to the end of the array literal?

Reply:
Done, though this required a small adjustment to the implementation of ES.DefaultTemporalLargestUnit to compile again (though I think this change would be needed anyways as the existing implementation was not renaming-safe for Closure Compiler).